Documentation/metrics.md

```diff
@@ -12,64 +12,34 @@ The naming of metrics follows the suggested [Prometheus best practices][promethe
 
 The metrics under the `etcd` prefix are for monitoring and alerting. They are stable high level metrics. If there is any change of these metrics, it will be included in release notes.
 
-### http requests
+Metrics that are etcd2 related are documented [here][v2_http_metrics].
 
-These metrics describe the serving of requests (non-watch events) served by etcd members in non-proxy mode: total 
-incoming requests, request failures and processing latency (inc. raft rounds for storage). They are useful for tracking
- user-generated traffic hitting the etcd cluster . 
+### gRPC requests
 
-All these metrics are prefixed with `etcd_http_`
+These metrics describe the requests served by a specific etcd member: total received requests, total failed requests and processing latency. They are useful for tracking user-generated traffic hitting the etcd cluster.
+
+All these metrics are prefixed with `etcd_grpc_`
 
 | Name                           | Description                                                                         | Type                   |
-|--------------------------------|-----------------------------------------------------------------------------------------|--------------------|
-| received_total                 | Total number of events after parsing and auth.                                      | Counter(method)        |
-| failed_total                   | Total number of failed events.                                                      | Counter(method,error)  |
-| successful_duration_second     |  Bucketed handling times of the requests, including raft rounds for writes.          | Histogram(method)      |
+|--------------------------------|-------------------------------------------------------------------------------------|------------------------|
+| received_total                 | Total number of received requests                                                   | Counter(method)        |
+| failed_total                   | Total number of failed requests.                                                    | Counter(method,error)  |
+| successful_duration_second     | Bucketed handling duration of the requests.                                         | Histogram(method)      |
```

Comment:
We're coming from the other direction, if there's increased latency due to failures timing out and you are only exporting successes how would you notice it? For servers where failures are an abnormal situation which warrant alerting, it thus makes sense to include all requests in the latency as increased latency due to either latency or slow failures will be obvious. Only if failures were normal would you consider splitting it out.